src/main/java/frc/robot/commands/ShooterEject.java

```diff
@@ -3,6 +3,8 @@
 
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.shuffleboard.EventImportance;
```

Comment:
Looks like this isn't used?

Reply:
True, I think I had something in there originally and needed those but now I have deleted them as they are no longer necessary.